src/monocle_apptrace/metamodel/entities/README.md

```diff
@@ -1,33 +1,44 @@
 # Monocle Entities
 The entity type defines the type of GenAI component that Monocle understand. The monocle instrumentation can extract the relevenat information for this entity. There are a fixed set of [entity types](./entity_types.py) that are defined by Monocle out of the box, eg workflow, model etc. As the GenAI landscape evolves, the Monocle community will introduce a new entity type if the current entities won't represent a new technology component.
 
-## Entity Types
+## Entities
 Following attributes are supported for all entities
 | Name | Description | Required |
 | - | - | - |
 | name | Entity name generated by Monocle | Required |
-| type | Monocle Entity type | True |
+| type | entity.category.type | True |
 
 ### MonocleEntity.Workflow
-Workflow ie the core application code. Supported types are -
+Workflow ie the core application code. 
+#### Workflow types
 - generic
 - langchain
 - llama_index
```

Comment:
llamaindex is a single word as seen on their official page
https://www.llamaindex.ai/